lib/elixir/src/elixir_erl.erl

```diff
@@ -515,14 +515,29 @@ supports_extra_chunks_option() ->
 docs_chunk(Set, Line, Def, Defmacro, Types, Callbacks) ->
   case elixir_compiler:get_opt(docs) of
     true ->
-      ChunkData = term_to_binary({elixir_docs_v1, [
+      {DocLine, Doc} = get_moduledoc(Line, Set),
+
+      FunctionAndMacroDocs = get_docs(Set, Def, Defmacro),
+      CallbackDocs = get_callback_docs(Set, Callbacks),
+      TypeDocs = get_type_docs(Set, Types),
+
+      DocsChunkData = term_to_binary({docs_v1,
+        erl_anno:new(DocLine),
```

Comment:
@josevalim do we want to include the file in this?

Reply:
Just the line if fine for now IMO.